share/migrations/0030_auto_20170425_1427.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
```

Comment:
I did, the only changes are Meta options and setting `related_name='versions'` on all the `persistent_ids`. 1.11 (or maybe 1.10, I forget) was unhappy with the `versions` manager on model instances, and when I was halfway through trying to rehack it I realized there was no reason not to just use the related manager instead.